### ISSUE TITLE
Origins compatibility

### ISSUE THREAD

**Issue by ParasiticSquid** [Pokecube-Development/Pokecube-Issues-and-Wiki#1130]
**Is your feature request related to a problem? Please describe.**
The ability to link origins to pokecube to set a origin pokecube type or a origin to have a pokecube ability.

**Describe the solution you'd like**
A way for players/origin addon or even pokecube datapack makers to set a pokecube type in the origins properties and have pokecube typed moves effect the origin player based on the set type, doing more or less damage based on what type/types the origin is set too. Basic examples being: Avian as flying type, phantom as ghost type, merling as water type, arachnid as bug type, feline as normal type, blazeborn as fire type, enderian as psychic type, elytrian as flying type, and shulk as psychic/rock type. 

The ability to also set a pokecube ability as a origin ability would also be useful. For example, setting one of a origins ability to use a pokecube one such a pickup finding items similar to being used on pokemobs but placing them directly in the players inventory or the abilities flash fire, water absorb or volt absorb heal the player when they touch certain things such as lava, water, or possibly even electical wires from immersive engineering. Getting hit by another player or pokemon with static, flame body, or poison point would cause the attacking player/pokemon to become poisoned, burned, or paralyzed. Essentially just pokemon abilities but allowing people to set origins to use them as a ability too if possible to do so.

**Describe alternatives you've considered**
There aren't really any to my knowledge.

**Additional context**
Basically depending on the pokecube moves/pokemon attack moves do more or less damage for better immersion when using both mods as well as allowing origins to use pokecube abilities where they would make sense. A big example being phantom not being able to be hit by normal or fighting moves and feline not being able to be hit by ghost moves but super effective and not very effective applies across the board where needed. I don't see any easy way to allow origins to use moves but at least the other two features, or even just the typing one, could add a lot of function when  both mods are used together. Could also be a basic stop gap for players wanting the poke transformer back and dedicated enough to set up origins to use such abilities or types as needed.

**Comment by Thutmose** [comment]
Can in include a link to this "origins"?
